Merge_Sort_3_Way/Merge_Sort_3_Way.c

```diff
@@ -0,0 +1,123 @@
+#include<stdio.h>
+//#include<stdlib.h>
+
+void mergesort(long int *arr1, int p , int q ,long int *arr2);
+int merge(long int *arr1,int l,int m1, int m2, int h, long int *arr2 );
+
+int main()
+{ 
+
+  int length;
+
+  long int n;
+  printf("Number of elements in the array : ");
+  scanf("%ld",&n);
+  
+  long int* arr1 = (long int *)malloc(n * sizeof(long int));
+  long int* arr2=(long int *)malloc (n * sizeof(long int ));
```

Comment:
- Add space before and after equals sign.
- Remove space after `malloc`
- Remove space after the last `long int`